src/Compilers/CSharp/Test/EndToEnd/EndToEndTests.cs

```diff
@@ -13,8 +13,14 @@
 using Microsoft.CodeAnalysis.CSharp.Symbols;
 using Microsoft.CodeAnalysis.PooledObjects;
 
-namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Emit
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.EndToEnd
 {
+    /// <summary>
+    /// The EndToEnd tests are isolated from other compiler test to avoid other tests and random
```

Comment:
Rather than a doc comment, consider putting a README.md in the directory that details this. It shows up better when looking in GitHub. 

There we can detail all of the elements we are trying to isolate because they can impact our stress points. Specifically: 

1. JIT ordering 
2. GC side effects
3. Implicit caching
4. Starting stack size